skdownscale/pointwise_models/quantile.py

```diff
@@ -127,10 +121,8 @@ def transform(self, X):
 
         # do the final mapping
         qt_kws = default_none_kwargs(self.qt_kwargs, copy=True)
-        if 'n_quantiles' not in qt_kws:
-            qt_kws['n_quantiles'] = len(X)
 
-        x_quantiles = quantile_transform(x_to_cdf, copy=True, **qt_kws)
+        x_quantiles = CunnaneTransformer(**qt_kws).fit(x_to_cdf).transform(x_to_cdf)
```

Comment:
use `fit_transform()` here

Reply:
So should this line be?
`x_quantiles = CunnaneTransformer(**qt_kws).fit_transform()(x_to_cdf)`

Reply:
I don't think so. The standard api is here: https://github.com/scikit-learn/scikit-learn/blob/baf828ca1/sklearn/base.py#L824